travis-ci/vmtest/run_selftests.sh

```diff
@@ -1,11 +1,22 @@
 #!/bin/bash
 
-set -eux
+set -euxo pipefail
 
-mount -t bpf bpffs /sys/fs/bpf
-mount -t debugfs none /sys/kernel/debug/
+test_progs() {
+	echo TEST_PROGS
+	./test_progs ${BLACKLIST:+-b$BLACKLIST} ${WHITELIST:+-t$WHITELIST}
+}
 
-ip link set lo up
+test_maps() {
+	echo TEST_MAPS
+	# Allow failing on older kernels.
+	./test_maps || [ ${KERNEL} != 'LATEST' ]
+}
+
+test_verifier() {
+	echo TEST_VERIFIER
+	./test_verifier
+}
```

Comment:
let's not just run test_maps and test_verifier not on latest. They have no built-in mechanisms to blacklist latest tests, so we won't be able to rely on them for older kernel anyway. So why bother and prolong tests.